test/crdsvalidation/kongpluginbindings/kongpluginbindings_test.go

```diff
@@ -28,11 +28,15 @@ func TestKongPluginBindings(t *testing.T) {
 				tc := tc
 				t.Run(tc.Name, func(t *testing.T) {
 					cl := cl.KongPluginBindings(tc.KongPluginBinding.Namespace)
-					kpb, err := cl.Create(ctx, &tc.KongPluginBinding, metav1.CreateOptions{})
+					entity, err := cl.Create(ctx, &tc.KongPluginBinding, metav1.CreateOptions{})
 					if err == nil {
 						t.Cleanup(func() {
-							assert.NoError(t, client.IgnoreNotFound(cl.Delete(ctx, kpb.Name, metav1.DeleteOptions{})))
+							assert.NoError(t, client.IgnoreNotFound(cl.Delete(ctx, entity.Name, metav1.DeleteOptions{})))
 						})
+						// Create doesn't set the status, so we need to update it explicitly.
```

Comment:
Why doesn't it? isn't the programmed condition defaulted?

Reply:
It defaults to unknown: https://github.com/Kong/kubernetes-configuration/blob/e817f2e1ac12516e0cf5d9f93e887137e0fe3c55/api/configuration/v1alpha1/kongpluginbinding_types.go#L171-L172

I've copy pasted this change from `KongService` and `KonnectControlPlane` tests in case we add validation for `KongPluginBinding` that relies on status. Otherwise the status is not updated.